uninstall.php

```diff
@@ -37,6 +37,5 @@
  * @since 1.4.0
  */
 function perflab_delete_plugin_option() {
-	delete_option( 'perflab_modules_settings' );
```

Comment:
Should this remain? What if someone was on Performance Lab v2 and then they upgrade to v3 but then uninstall. Won't this option be then remaining? Perhaps leave this line here but add a comment to say it was from an older version. 

That brings to mind that it seems at present there is an _autoloaded_ `perflab_modules_settings` option which is getting loaded needlessly. It seems there should be an upgrade routine added for 2.9.0 to 3.0.0 which deletes this option or at least makes it not be autoloaded.

Reply:
Oh, this may be what @thelovekesh mentioned in https://github.com/WordPress/performance/pull/1060#issuecomment-2009992383.

Reply:
Heh, and @felixarntz in https://github.com/WordPress/performance/pull/1060#issuecomment-2010175828.

Reply:
@westonruter Yeah, I don't think we need to keep it if we instead have a "migration" logic that deletes this option after an update. That way the old option is only referenced in one place, and it makes more sense to delete it after the update rather than only once (or rather if) the plugin is uninstalled.